src/Sylius/Bundle/CoreBundle/Form/Type/ImageType.php

```diff
@@ -14,7 +14,7 @@
 use Sylius\Bundle\ResourceBundle\Form\Type\AbstractResourceType;
 use Symfony\Component\Form\FormBuilderInterface;
 
-class ImageType extends AbstractResourceType
+abstract class ImageType extends AbstractResourceType
```

Comment:
Can't we have 3 types? `sylius_image`, `sylius_product_variant_image` and `sylius_taxon_image`?